src/opm/output/eclipse/AggregateNetworkData.cpp

```diff
@@ -486,7 +490,7 @@ void staticContrib(const Opm::Schedule&         sched,
     auto uptr_nd_res = findInVector<std::string>(nodeNames, branch.uptree_node());
     iBran[Ix::UpTreeNode] = (uptr_nd_res) ? uptr_nd_res.value() + 1 : 0 ;
 
-    iBran[Ix::VfpTableNo] = (branch.vfp_table().has_value()) ? branch.vfp_table().value() : 0;
```

Comment:
Under what circumstance, `branch.vfp_table().has_value()` will be `false`?

Reply:
Not sure, assumed there were cases since it was defined as an optional - if not we should probably change this to an int?